tutorials/dynamic-output-topic/kstreams.html

```diff
@@ -0,0 +1,6 @@
+---
+layout: tutorial
+permalink: /streams-dynamic-output-topic/kstreams
```

Comment:
In case we add other flavors of this tutorial in the future, let's remove streams from the slug/permalink

```suggestion
permalink: /dynamic-output-topic/kstreams
```